CMakeLists.txt

```diff
@@ -117,7 +128,7 @@ if(LOGROTATE_HAS_SU)
   set(LOGROTATE_USE_SU "\n\tsu ${ICINGA2_USER} ${ICINGA2_GROUP}")
 endif()
 
-find_package(Boost ${BOOST_MIN_VERSION} COMPONENTS thread system program_options regex REQUIRED)
+find_package(Boost ${BOOST_MIN_VERSION} COMPONENTS filesystem thread system program_options regex REQUIRED)
```

Comment:
We shouldn't introduce a new boost dependency. The existing code for filesystem operations is already there.

Reply:
There's really nothing wrong with that IMO.

Reply:
I think we should actually prefer moving file tests to `boost::filesystem`